context/app/static/js/components/workspaces/WorkspacesAuthGuard/WorkspacesAuthGuard.tsx

```diff
@@ -1,37 +1,144 @@
-import React, { ReactNode } from 'react';
-import { styled } from '@mui/material/styles';
+import React, { PropsWithChildren, ReactNode } from 'react';
+import Stack from '@mui/material/Stack';
+import Typography from '@mui/material/Typography';
+import Button from '@mui/material/Button';
+import Paper from '@mui/material/Paper';
 
 import { useAppContext } from 'js/components/Contexts';
-import Description from 'js/shared-styles/sections/Description';
-import { InternalLink } from 'js/shared-styles/Links';
 import ContactUsLink from 'js/shared-styles/Links/ContactUsLink';
+import { Alert } from 'js/shared-styles/alerts';
+import { InternalLink } from 'js/shared-styles/Links';
+import OutboundIconLink from 'js/shared-styles/Links/iconLinks/OutboundIconLink';
+
+function ContactUsForAccess() {
+  return (
+    <>
+      <ContactUsLink>Contact us</ContactUsLink> for questions about access.
```

Comment:
I've run into this exact situation before where we need to provide custom `Contact us` text to stick to sentence-case instead of capitalizing the `Us` - should we just add a boolean prop for e.g. `sentenceCase`?